### PR TITLE
Add HSTS header

### PR DESCRIPTION
**Pull request by openjck**

### CODE REVIEW

**Comment by openjck** [comment]
Updated! I even gave you credit for the `SECURE_PROXY_SSL_HEADER` thing. :smile:

File: shielddash/settings.py [hunk 1]
```diff
@@ -111,6 +111,8 @@
 
 STATIC_URL = '/static/'
 
+SECURE_SSL_REDIRECT = not DEBUG
+SECURE_HSTS_SECONDS = (60 * 60 * 7 * 24 * 365)
```
Comment (robhudson) [inline shielddash/settings.py]: That's 7 years. Originally you had 1 year so I'd drop the 7. I think 1 year is good.

Comment (openjck) [inline shielddash/settings.py]: Heh, bad math. Updated!

**Comment by robhudson** [comment]
r+ with comment and passing tests. 👍

**Comment by openjck** [comment]
Added `config('SSL', False)`. I also added the ENVVAR on Heorku and set it to `True`.

**Pull request merged**